pkg/download/handler.go

```diff
@@ -51,3 +51,36 @@ func RegisterHandlers(r *mux.Router, opts *HandlerOpts) {
 	r.Handle(PathVersionModule, LogEntryHandler(ModuleHandler, opts)).Methods(http.MethodGet)
 	r.Handle(PathVersionZip, LogEntryHandler(ZipHandler, opts)).Methods(http.MethodGet)
 }
+
+func RegisterWebhookHandlers(r *mux.Router, opts *HandlerOpts) {
```

Comment:
I think the `download` package should be strictly for the download protocol...so this function and its handlers can probably live in their own package called `webhooks` either in the `actions` directory or in the `pkg` directory. Or you might want to flatten it out and just put it in the `actions` dir directly.

Reply:
Good point. I'll push a change that moves this to another package!